docs/openapi/resources/presentation-submissions.yml

```diff
@@ -4,7 +4,6 @@ post:
   description: End a presentation exchange flow
   tags:
     - Presentations
```

Comment:
```suggestion
    - Presentations
        security:
            - OAuth2: [submit:presentations]
```

might be more inline with the spec?

Reply:
@mprorock, `/presentations/available` and `/presentations/submission` MUST NOT require OAuth2, see #242 and #243.

Reply:
+1 switching to approved - my bad